components/hidden-text/src/index.js

```diff
@@ -23,8 +24,9 @@ const StyledSummary = styled('summary')({
     color: LINK_HOVER_COLOUR,
   },
   ':focus': {
-    outline: `${FOCUS_WIDTH} solid ${FOCUS_COLOUR}`,
-    outlineOffset: '-1px',
+    outline: `calc(${FOCUS_WIDTH} + 1px) solid ${FOCUS_COLOUR}`,
```

Comment:
Shame to have to use calc here for this, I thought maybe the calculation could be done with JS and the result used here, but you will have to strip the `px` off the values and then re-add them... bit of a schlep?